forge-gui/res/cardsfolder/upcoming/alaundo_the_seer.txt

```diff
@@ -0,0 +1,15 @@
+Name:Alaundo the Seer
+ManaCost:2 G U
+Types:Legendary Creature Human Shaman
+PT:3/5
+A:AB$ Draw | Cost$ T | SubAbility$ DBExile | StackDescription$ SpellDescription | SpellDescription$ Draw a card, then exile a card from your hand and put a number of time counters on it equal to its mana value. It gains "When the last time counter is removed from this card, if it's exiled, you may cast it without paying its mana cost. If you cast a creature spell this way, it gains haste until end of turn." Then remove a time counter from each other card you own in exile.
+SVar:DBExile:DB$ ChangeZone | Origin$ Hand | Destination$ Exile | TgtPrompt$ Select a card to exile with a number of time counters equal to its mana value | SubAbility$ DBPutTimeCounters | RememberChanged$ True | Mandatory$ True
```

Comment:
you can use `WithCountersType` and `WithCountersAmount` here to eliminate line 7

Reply:
I tried that and it didn't work. I then tried following code execution using IntelliJ's debugger and it never goes through a section of code that uses `WithCountersType` or `WithCountersAmount`. 

I could add this to my PR. Specifically, on line 1388 of `ChangeZoneEffect.java`, I could add code that's similar to lines 775-779, something like:
```
if (sa.hasParam("WithCountersType")) {
    CounterType cType = CounterType.getType(sa.getParam("WithCountersType"));
    int cAmount = AbilityUtils.calculateAmount(hostCard, sa.getParamOrDefault("WithCountersAmount", "1"), sa);
    movedCard.addCounter(cType, cAmount, player, counterTable);
}
```

What do you think?

Reply:
Oh right, because this is in changeZonePlayerInvariant

Good call

ChangeZoneEffect is kind of a tangle

I wonder if we could/should add a void for the counter stuff in ChangeZoneEffect to avoid having the same code in 2-3 (more?) different spots

Thoughts @tool4ever @Hanmac ??

My expectation is that the others will agree it's better to add the params to ChangeZone than using a PutCounter line, but we'll see 😁